app/src/tests/components/contents/manageAccess/updatePermissions/utils/convertUpdatePermissionsForFetch.test.ts

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable max-len */
 import {
-  NewRolePermission,
+  BaseRolePermission,
   RolePermission
```

Comment:
Unused import warning for RolePermission here. Couple of similar gha linting warnings on UpdateRoleDropDownMenu.test.tsx